client/src/components/QnA.jsx

```diff
@@ -8,6 +8,17 @@ import AddQ from './QnAComponents/AddQ.jsx';
 const QnAContainer = styled.div`
   display: flex;
   flex-direction: column;
+  width: 70vw;
```

Comment:
You'll just want to update this to 65 (and then I think that'll require a change to your search bar stuff)